src/push/monitor.ts

```diff
@@ -94,6 +95,12 @@ export async function createLightweightMonitors(
       lwFiles.add(abs);
     }
   });
+  // Warn users about schedule that are less than 60 seconds
+  if (lwFiles.size > 0) {
```

Comment:
This shows always, even when a user doesn't have a 30s interval. Is that desired?

Reply:
Yeah its hard to do it per monitor and also bubbling up that error to the overall monitor creation logic. Kept it here as its simpler and also serves as a warning.